src/components/Label/Label.scss

```diff
@@ -95,12 +95,48 @@ $transitionTimingFunction: ease-in-out;
             &#{$block}_style {
                 &_default,
                 &_rounded {
-                    border-radius: 4px;
+                    border-radius: var(--yc-border-radius-s);
```

Comment:
For size `xs` used `border-radius-s`

Reply:
In design XS Label has same border radius 5px as S. I added another variable with equal value